benchmarks/particle_integration_scheme/rk4_38.prm

```diff
@@ -0,0 +1,5 @@
+subsection Postprocess
```

Comment:
since you add this option to the `particle_integration_scheme` benchmark, please also add the results and maybe a discussion to the documentation of this benchmark.